app/views/metadata_presenter/header/show.html.erb

```diff
@@ -26,7 +26,12 @@
         </span>
       </a>
     </div>
-    <% if service.service_name.present? %>
+    <% if use_external_start_page? && service.service_name.present? %>
```

Comment:
Would it make sense instead of this if-else, having a helper method, for instance `#start_page_url`, that returns the external url if `use_external_start_page?` otherwise returns `root_path`?
As the markup etc is the same and only the url changes, to avoid repetition. Just a suggestion!